docs/querying/examples.md

```diff
@@ -95,3 +95,25 @@ Assuming this metric contains one time series per running instance, you could
 count the number of running instances per application like this:
 
     count by (app) (instance_cpu_time_ns)
+  
+To find out the maximum value of a metric over a period of time, for eg. last 
+6 hours, you can query like this:
+
+    max_over_time(instance_cpu_time_ns[6h])
```

Comment:
This is not good query, the metric isn't following bestpractices for naming and it looks like a counter.

Reply:
I don't understand. its the same metric used in other examples also. 

> it looks like a counter
Why do you think so?

Reply:
Hmm, we should fix those examples then. This sort of data is usually exposed as a counter - which the other examples show.